index.html

```diff
@@ -29,22 +34,19 @@ <h2 class="first-name software-dev">I'm a software developer</h2>
       <li><a href="https://github.com/stephnna"><img class="social-img" src="images/github.png" alt="github"></a></li>
       <li><a href=""><img class="social-img" src="images/angellist.png" alt="angellist"></a></li>        
   </ul>
-  <div class="arrow-flex display-flex">
-    <span class="arrow">&or;</span>
-</div>
-</section>
-
+  <img class="down-arrow" src="./images/down-arrow.svg" alt="down-arrow">
+</section>  
+  </header> 
+  <main>
 <!-- Recent Work Session -->
-  <h2 class="recent-work-align">My Recent</h2>
-  <h2 class="recent-work-align">Works</h2>
+  <h2 class="recent-work-align">My Recent<br class="break"> Works</h2>
 <div class="dash-flex display-flex">
 <span class="dash">&#8212;</span>
 </div>
 <section class="container-recent">  
   <div class="grid-project">
     <div class="multi-post">
```

Comment:
- [ ] Let's try to follow the design guidelines shown below. On desktop versions between 768px and 1080, the Projects section is cut off on the right instead of being centered. Please fix it 🙏

### Your Design
![image](https://user-images.githubusercontent.com/46839548/166009499-d1101754-6132-4fa8-b2a6-a22de2db02d5.png)
### Figma Design
![image](https://user-images.githubusercontent.com/46839548/166009863-f680c6c5-6f7a-4e39-a130-41e4cd89e2bb.png)